examples/with-typescript-graphql/lib/apollo.ts

```diff
@@ -16,7 +17,13 @@ export type ResolverContext = {
 function createIsomorphLink(context: ResolverContext = {}) {
   if (typeof window === 'undefined') {
     const { SchemaLink } = require('@apollo/client/link/schema')
-    const { schema } = require('./schema')
+    const { readFileSync } = require('fs')
+    const { makeExecutableSchema } = require('@graphql-tools/schema')
+
+    const schema = makeExecutableSchema({
+      typeDefs: readFileSync('lib/schema.graphql', 'utf8'),
```

Comment:
worth to check this on a vercel deployment. readFileSync is highly dependent on cwd